src/module.indexer/model/token.ts

```diff
@@ -25,17 +26,28 @@ export class TokenIndexer extends Indexer {
 
           const data: TokenCreate = (stack[1] as OP_DEFI_TX).tx.data
 
-          let dctId = await this.dctMapper.getLatest()
+          // get the latest tokenId via rpc listTokens
+          const tokens = await this.client.token.listTokens()
+          if (Object.keys(tokens).length === 0) {
```

Comment:
I think can reduce some codes here, effectively they behave the same.
```suggestion
          const token = tokens[data.symbol]
          if (token === undefined) throw new RpcNotFoundIndexerError('listTokens', data.symbol)
```

Reply:
yea that's an ideal way but that has design issue, the resp is out of expected
```
{ '0': ...token, '1', ...token }
```